middlewares/validators/user.js

```diff
@@ -260,17 +260,18 @@ const validateImageVerificationQuery = async (req, res, next) => {
 };
 
 async function validateUpdateRoles(req, res, next) {
-  const schema = joi.object().strict().min(1).max(1).keys({
+  const schema = joi.object().strict().min(1).max(2).keys({
+    // either member or archived with reason (optional) is allowed
```

Comment:
Is it important?

Reply:
yes, if we give max value `2` then only validation will accept two keys i.e. `archived` with `reason`